std/node/_fs/_fs_mkdtemp.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
```

Comment:
If this contains code from Node probably add that copyright? (there is already a process for that I think?)